scripts/build-durable-nonce.js

```diff
@@ -0,0 +1,113 @@
+const web3 = require('@solana/web3.js');
+const { binary_to_base58, base58_to_binary } = require('base58-js');
+const { createInterface } = require('readline');
+const { program } = require('commander');
+
+const readline = createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const connection = new web3.Connection(
+  web3.clusterApiUrl('devnet'),
+  'confirmed'
+);
+
+main();
+
+function question(question) {
+  return new Promise(resolve => {
+    readline.question(question, data => {
+      readline.close();
+      resolve(data);
+    });
+  });
+}
+
+async function createNonceAccount(wallet) {
+  // Generate keypair for nonce account
+  let nonceAccount = web3.Keypair.generate();
+
+  // Get Minimum amount for rent exemption
+  let minimumAmount = await connection.getMinimumBalanceForRentExemption(
+    web3.NONCE_ACCOUNT_LENGTH
+  );
+
+  // create nonce account transaction
+  let nonceAccountTransaction = new web3.Transaction().add(
+    web3.SystemProgram.createNonceAccount({
+      fromPubkey: wallet.publicKey,
+      noncePubkey: nonceAccount.publicKey,
+      authorizedPubkey: wallet.publicKey,
+      lamports: minimumAmount,
+    })
+  );
+
+  await web3.sendAndConfirmTransaction(connection, nonceAccountTransaction, [
+    wallet,
+    nonceAccount,
+  ]);
+
+  return nonceAccount;
+}
+
+async function handleImport(network) {
+  if (network !== 'devnet' && network !== 'mainnet-beta') {
```

Comment:
here's how you handle network from the comment below
```suggestion
async function handleImport({ network }) {
  if (network !== 'devnet' && network !== 'mainnet-beta') {
```

Reply:
doesn't commander already handle invalid arguments for you? if so, you should be able to remove these constraints.